policyengine_us/parameters/gov/states/de/tax/income/subtractions/elderly_disabled/subtraction_result.yaml

```diff
@@ -0,0 +1,23 @@
+description: Delaware provides an exclusion to filers with income subtractions below this threshold.
+JOINT:
+  2021-01-01: 20_000
+HEAD_OF_HOUSEHOLD:
+  2021-01-01: 10_000
+WIDOW:
+  2021-01-01: 10_000
+SINGLE:
+  2021-01-01: 10_000
+SEPARATE:
+  2021-01-01: 10_000
+metadata:
+  reference:
+    - title: "Delaware individual income tax instructions for 2022 Page 7 Line 11"
+      href: https://revenuefiles.delaware.gov/2021/PIT-RES_TY21_2021-01_Instructions.pdf
+    - title: "Delaware individual income tax instructions for 2021 Page 7 Line 11"
+      href: https://revenuefiles.delaware.gov/2021/PIT-RES_TY21_2021-01_Instructions.pdf
+    - title: Delaware State Taxes, Part II, Chapter 11, Subchapter 2, § 1106. Modifications,(b), (2) 
+      href: https://delcode.delaware.gov/title30/c011/sc02/index.html
+  unit: year
+  label: Delaware exclusion income subtraction amount
```

Comment:
```suggestion
  label: Delaware persons 60 or over or disabled exclusion subtraction amount threshold
```

Reply:
other labeles similar

Reply:
Labels should not include numbers (parameter values). Replace "persons 60 or over" with "aged" or "elderly".